src/antares/service/local_services/thermal_local.py

```diff
@@ -31,7 +39,74 @@ def update_thermal_properties(
+        thermal_clusters = []
+        if thermal_dict:
+            for thermal_cluster in thermal_dict:
+                thermal_properties = ThermalClusterPropertiesLocal(
+                    group=thermal_dict[thermal_cluster]["group"],
+                    thermal_name=thermal_dict[thermal_cluster]["name"],
+                    enabled=thermal_dict[thermal_cluster]["enabled"],
+                    unit_count=thermal_dict[thermal_cluster]["unitcount"],
+                    nominal_capacity=thermal_dict[thermal_cluster]["nominalcapacity"],
+                    gen_ts=thermal_dict[thermal_cluster]["gen-ts"],
+                    min_stable_power=thermal_dict[thermal_cluster]["min-stable-power"],
+                    min_up_time=thermal_dict[thermal_cluster]["min-up-time"],
+                    min_down_time=thermal_dict[thermal_cluster]["min-down-time"],
+                    must_run=thermal_dict[thermal_cluster]["must-run"],
+                    spinning=thermal_dict[thermal_cluster]["spinning"],
+                    volatility_forced=thermal_dict[thermal_cluster]["volatility.forced"],
+                    volatility_planned=thermal_dict[thermal_cluster]["volatility.planned"],
+                    law_forced=thermal_dict[thermal_cluster]["law.forced"],
+                    law_planned=thermal_dict[thermal_cluster]["law.planned"],
+                    marginal_cost=thermal_dict[thermal_cluster]["marginal-cost"],
+                    spread_cost=thermal_dict[thermal_cluster]["spread-cost"],
+                    fixed_cost=thermal_dict[thermal_cluster]["fixed-cost"],
+                    startup_cost=thermal_dict[thermal_cluster]["startup-cost"],
+                    market_bid_cost=thermal_dict[thermal_cluster]["market-bid-cost"],
+                    co2=thermal_dict[thermal_cluster]["co2"],
+                    nh3=thermal_dict[thermal_cluster]["nh3"],
+                    so2=thermal_dict[thermal_cluster]["so2"],
+                    nox=thermal_dict[thermal_cluster]["nox"],
+                    pm2_5=thermal_dict[thermal_cluster]["pm2_5"],
+                    pm5=thermal_dict[thermal_cluster]["pm5"],
+                    pm10=thermal_dict[thermal_cluster]["pm10"],
+                    nmvoc=thermal_dict[thermal_cluster]["nmvoc"],
+                    op1=thermal_dict[thermal_cluster]["op1"],
+                    op2=thermal_dict[thermal_cluster]["op2"],
+                    op3=thermal_dict[thermal_cluster]["op3"],
+                    op4=thermal_dict[thermal_cluster]["op4"],
+                    op5=thermal_dict[thermal_cluster]["op5"],
+                    cost_generation=thermal_dict[thermal_cluster]["costgeneration"],
+                    efficiency=thermal_dict[thermal_cluster]["efficiency"],
+                    variable_o_m_cost=thermal_dict[thermal_cluster]["variableomcost"],
+                )
+
+            thermal_clusters.append(
```

Comment:
Put this inside the loop, it's not indented correctly

Reply:
Ok, for the last commit